package.json

```diff
@@ -154,6 +154,7 @@
     "nock": "^9.3.3",
     "nyc": "^14.0.0",
     "prettier": "1.14.3",
+    "shx": "^0.3.2",
```

Comment:
I'm game for this, but I would recommend [rimraf](https://www.npmjs.com/package/rimraf) (smaller, more used, less other dependencies).

The script then becomes `rimraf lib` (untested - lol)